packages/core/src/internal/utils.js

```diff
@@ -39,12 +39,18 @@ export const is = {
 }
 
 export const object = {
-  assign(target, source) {
-    for (let i in source) {
-      if (hasOwn(source, i)) {
-        target[i] = source[i]
+  assign: Object.assign || function (target) {
```

Comment:
this actually won't work as expected. Using `Object.assign` works fine, but `for .. in` doesn't iterate through symbols

Reply:
1 @Andarist that was done intentionally, since if you use Symbol, you have Object.assign defined in your browser. and I didn't want to add additional check for `Object.getOwnPropertySymbols` existence
2 implementation was borrowed from babel https://github.com/babel/babel/blob/2abd7839e16e3fa63fc381e3ec6ddd83efdf703e/packages/babel-helpers/src/helpers.js#L359

Reply:
> @Andarist that was done intentionally, since if you use Symbol, you have Object.assign defined in your browser. and I didn't want to add additional check for Object.getOwnPropertySymbols existence

In world with polyfills that is unfortunately not true in all the cases. You can polyfill Symbols and skip polyfilling `Object.assign`.